app/models/record.server.ts

```diff
@@ -0,0 +1,98 @@
+import type { Record, User } from '@prisma/client';
+import { prisma } from '../db.server';
+
+export const createRecordInDB = async (
+  username: User['username'],
+  type: Record['type'],
+  name: Record['name'],
+  value: Record['value'],
+  createdAt: Record['createdAt'],
+  updatedAt: Record['updatedAt'],
+  expiresAt: Record['expiresAt'],
+  status: Record['status']
+) => {
+  return await prisma.record.create({
+    data: {
+      username,
+      type,
+      name,
+      value,
+      createdAt,
+      updatedAt,
+      expiresAt,
+      status,
+    },
+  });
+};
+
+export const findRecordById = async (id: Record['id']) => {
```

Comment:
Isn't this the same as what you have below for `getRecordFromDB()`?

Reply:
Will be removed